src/Forms/Main.cs

```diff
@@ -19228,7 +19228,7 @@ private Paragraph InsertNewTextAtVideoPosition()
 
             // prevent overlap
             var endTotalMilliseconds = videoPositionInMilliseconds + Configuration.Settings.General.NewEmptyDefaultMs;
-            var next = _subtitle.Paragraphs[index];
+            var next = _subtitle.Paragraphs.ElementAtOrDefault(index);

```

Comment:
SE uses `_subtitle.GetParagraphOrDefault`

Reply:
And I messed up the word "Paragraph" in the new commit. smh

Reply:
This is better than `count > index`, right?

Reply:
Let me know if it's better to close this and open a new one.
Or you might wanna do the commit yourself and close this. :D